crypto/keys/pem_signer_factory_test.go

```diff
@@ -21,6 +21,16 @@ import (
 )
 
 func TestPEMSignerFactory(t *testing.T) {
+	key, err := NewFromPrivatePEMFile("../../testdata/log-rpc-server.privkey.pem", "towel")
```

Comment:
We could, but the rest of the tests read that same file so it doesn't buy us much. Happy to make this change if you prefer though.